src/test/java/org/assertj/core/api/recursive/comparison/RecursiveComparisonConfiguration_multiLineDescription_Test.java

```diff
@@ -331,6 +331,7 @@ void should_show_a_complete_multiline_description() {
                "  - java.lang.Double -> DoubleComparator[precision=1.0E-15]%n" +
                "  - java.lang.Float -> FloatComparator[precision=1.0E-6]%n" +
                "  - java.lang.Integer -> AbsValueComparator%n" +
+               "  - java.nio.file.Path -> INSTANCE%n" +
```

Comment:
it would be better to have a description instead of `INSTANCE`, like `naturalComparator` or even better: `lexicographic comparator (Path naturalComparator)`

Thw way to go is creating a Comparator class in order to override `toString`  as in `FloatComparator`

Reply:
Thanks, it should be better now.